cfpb/consumerfinance.gov
# Remove financial well-being content cruft

## Removals

- Remove "Latest blog posts" sidefoot section with entries from 2017 and earlier.
- Remove section linking to research reports from 2018 and earlier.


## Changes

- Properly indent meta elements.
- Minor formatting of line length.


## How to test this PR

1. Visit http://localhost:8000/consumer-tools/financial-well-being/ and see the blog posts are gone from the sidefoot.
1. Visit http://localhost:8000/consumer-tools/financial-well-being/about/ and see that the reports section is removed.


## Screenshots

Before
<img width="1199" alt="Screenshot 2024-01-09 at 1 33 41 PM" src="https://github.com/cfpb/consumerfinance.gov/assets/704760/f9449a58-5b4f-44d4-a678-33fcac556625">

After
<img width="1151" alt="Screenshot 2024-01-09 at 1 33 27 PM" src="https://github.com/cfpb/consumerfinance.gov/assets/704760/e00ef242-b9e1-4e9d-8a8a-789ccb484ba7">

Before
<img width="490" alt="Screenshot 2024-01-09 at 1 48 26 PM" src="https://github.com/cfpb/consumerfinance.gov/assets/704760/48a896e0-7df3-45a4-858a-9742da6f2d82">

After
<img width="440" alt="Screenshot 2024-01-09 at 1 48 14 PM" src="https://github.com/cfpb/consumerfinance.gov/assets/704760/f7b66a30-7c66-4e11-95b6-a22e37491907">
